examples/evaluate/04-TBROM_images.py

```diff
@@ -3,21 +3,26 @@
 3D field ROM example for images generation
 ------------------------------------------
 
-This example shows how PyTwin can be used to load and evaluate a Twin model in order to visualize ROM results in
-the form of images with predefined views. The script takes user inputs to evaluate the ROM and will display the
-corresponding image. A first image is generated using the point cloud based ROM Viewer embedded in the Ansys Digital
-Twin Runtime, and a second image is generated by loading the ROM results and performing the post processing on the
-CFD mesh by using PyFluent. Finally, the ROM results are post processing in a 3D Viewer enabled by PyFluent
+This example shows how PyTwin can be used to load and evaluate a twin model to
+visualize ROM results in the form of images with predefined views. The script
+takes the inputs to evaluate the ROM and displays the corresponding image.
+A first image is generated using the point cloud-based ROM Viewer embedded in
```

Comment:
I wasn't sure if "ROM Viewer" was a proper name from the GUI or should be changed to ROM viewer.

Reply:
ROM Viewer is a GUI name, see screenshot below:
![image](https://user-images.githubusercontent.com/108674439/218986370-6f61087e-8073-4ac7-b6e7-79f32918c2c5.png)
Here we are using the ROM Viewer embedded in the twin runtime, instead of the Stand Alone one shown in the screenshot. Does it make sense to you?

Reply:
@lboucin Makes sense, which means the existing capitalization is correct!